spec/javascripts/utilities/ReCaptchaV3.spec.tsx

```diff
@@ -0,0 +1,29 @@
+import { mount } from 'enzyme'
+
+import type { Props } from 'utilities/ReCaptchaV3'
+
+import type { FunctionComponent } from 'react'
+
+const ReCaptchaV3 = jest.requireActual('utilities/ReCaptchaV3').ReCaptchaV3 as FunctionComponent<Props>
+
+describe('ReCaptchaV3', () => {
```

Comment:
IMHO there's no need to "describe" the element you're in. The file is already "ReCaptchaV3". I would use describe to wrap different scenarios, for instance:
```js
describe("when recaptcha is disabled", () => {
  it('should do that...', () => {})
})

describe("when recaptcha is enabled", () => {
  it('should do this...', () => {})
})
```